packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -33,6 +33,7 @@ import {
   getTypeOfAxis,
   tooltipOfXAxislabels,
 } from '../../utilities/index';
+import { formatPrefix as d3FormatPrefix } from 'd3-format';
```

Comment:
In VerticalBarChart basic example, label 50k is showing outside the chart I think. Can you check.

Reply:
the label is above the 1st horizontal gridline but inside SVG